tests/start_test.py

```diff
@@ -1,17 +1,29 @@
 import unittest
-
```

Comment:
actually this line should not be deleted
there is always an empty line between three: internal packages (os, sys, unittest), downloaded packages (python-telegram-bot, for example), project packages (modules) - imports are in the stated order